demo/src/app/components/+datepicker/demos/custom-format/date-picker-custom-format.html

```diff
@@ -0,0 +1,19 @@
+<div class="row">
+  <div class="col-xs-12 col-12 col-md-4 form-group">
+    <form [formGroup]="myForm">
+      <input
+        class="form-control"
+        [minDate]="minDate"
+        [maxDate]="maxDate"
+        #dp="bsDatepicker"
+        bsDatepicker
+        [formControlName]="'myDate'"
```

Comment:
it's just a string setter `formControlName="myDate"`

Reply:
it's just a matter of codestyle, anyway I changed it as you suggested